### PR TITLE
Scm version6.x.x

### PR DESCRIPTION
**Pull request by veon82**
- New class RTCVersion
- Fix compatibility with RTC version >= 6.0.0

### CODE REVIEW

**Comment by WtfJoke** [comment]
I love that change! 

Would make sense to support multiple version with that check.
Would you kindly please add some tests for it, if its somehow possible.

File: rtcFunctions.py [hunk 1]
```diff
@@ -11,14 +11,26 @@
...
+        version = output[1].split()[1]
         return version
 
+    @staticmethod
+    def greaterequalthan(version2compare):
+        thisver = RTCVersion.getversion().split('.')
+        v2c = version2compare.split('.')
+        if thisver == v2c:
+            return True
+        for n,v in enumerate(thisver):
```
Comment (WtfJoke) [inline rtcFunctions.py]: can this part be simplified? Please change the variable n and v to something more expressive

Comment (veon82) [reply rtcFunctions.py]: I'll try, now I'm going to sleep cause it's 5:00am here.
Late night programming is never good :)

**Comment by gpflaum** [comment]
The `scm version` command doesn't show the RTC version in 6.0.1 and earlier, just plugin versions. The RTC version was added to the output in [6.0.2](https://www.ibm.com/support/knowledgecenter/SS2L6K_6.0.2/com.ibm.team.scm.doc/topics/version_ex.html).

File: rtcFunctions.py (hunk 1 above)
Comment (WtfJoke) [reply rtcFunctions.py]: yeah looks like we live in the same timezone. At 5.00am programming is not good anymore^^

**Comment by WtfJoke** [comment]
> The scm version command doesn't show the RTC version in 6.0.1 and earlier, just plugin versions

Oh... Thank you very much @gpflaum 

So I guess I cant merge this pull request, because it makes it incompatible with versions previous 6.0.1.

Do you agree @veon82 ?

EDIT: 
I just saw @gpflaum other comment here https://github.com/rtcTo/rtc2git/issues/105#issuecomment-354118307. I like both ideas, but version set by config would be more elegant.
I would set default version 5 and on 6 you could do something else.

**Comment by veon82** [comment]
I apologize, I should have looked pertinent issues. 
I agree on handling the version on config, I'll update the PR once you upgrade develop branch with this change, ok?

**Comment by WtfJoke** [comment]
> I'll update the PR once you upgrade develop branch with this change, ok?

Yes, thats fine

**Comment by zevanty** [comment]
I don't mean to take any credit, but I implemented the SCM version config option in #137. The SCM v6 code was directly pulled from @veon82's code, so all credits go to him.

**Comment by WtfJoke** [comment]
Config has been extended from #137. You can do your changes @veon82

**Comment by veon82** [comment]
Done.
Thanks @zevanty for your PR.

**Comment by zevanty** [comment]
And thank you for figuring out the majority of the work needed to get v6 working :)

**Comment by marcaurele** [comment]
I confirm that this is working in RTC 6.0.4. It would be nice to merge this PR into master already.

**Comment by WtfJoke** [comment]
Finally merged it. Thanks a lot @veon82.

Sorry for taking ages :S

**Pull request merged**